pkg/rpc/server/server.go

```diff
@@ -2266,7 +2260,7 @@ func (s *Server) packResponse(r *request.In, result interface{}, respErr *respon
 // logRequestError is a request error logger.
 func (s *Server) logRequestError(r *request.Request, jsonErr *response.Error) {
 	logFields := []zap.Field{
-		zap.Error(jsonErr.Cause),
+		zap.String("error", jsonErr.Data),
```

Comment:
Well, if we have `zap.String` now, maybe we can also make @532910 poco más feliz if we're to use the word `cause` instead of `error` here?